tools/vf-component-library/src/site/_includes/layouts/components.njk

```diff
@@ -283,7 +283,7 @@ This approach is best for bare-bones Nunjucks environments, such as precompiled
 %}{% endraw %}
                 </code></pre>
               </details>
-              {% set angularComponents = "vf-badge, vf-banner, vf-hero, vf-button, vf-blockquote" %}
+              {% set angularComponents = "vf-badge, vf-banner, vf-hero, vf-button, vf-blockquote, vf-tabs" %}
```

Comment:
should we have this change in Tabs angular PR?

Reply:
Added this change to the Tabs PR.